language/move-prover/tests/testsuite.rs

```diff
@@ -29,10 +29,22 @@ const INCONSISTENCY_TEST_FLAGS: &[&str] = &[
     "--dependency=../move-stdlib/modules",
     "--dependency=../diem-framework/modules",
     "--check-inconsistency",
+    "-a=Std=0x1",
+    "-a=DiemFramework=0x1",
+    "-a=DiemRoot=0xA550C18",
+    "-a=CurrencyInfo=0xA550C18",
+    "-a=TreasuryCompliance=0xB1E55ED",
+    "-a=VMReserved=0x0",
 ];
 const REGULAR_TEST_FLAGS: &[&str] = &[
     "--dependency=../move-stdlib/modules",
     "--dependency=../diem-framework/modules",
+    "-a=Std=0x1",
+    "-a=DiemFramework=0x1",
+    "-a=DiemRoot=0xA550C18",
+    "-a=CurrencyInfo=0xA550C18",
+    "-a=TreasuryCompliance=0xB1E55ED",
+    "-a=VMReserved=0x0",
```

Comment:
Is there someway to populate these with the stuff in diem-framework and mvoe-stdlib, just worried about changes overtime

Reply:
Once we have packages we can do this without introducing a Cargo dep on the diem side of things. However, if I try to use the stuff from the diem-framework that will then introduce a diem dependency. So at least for now, I think we should probably keep this like this. I'll add a TODO and tag myself in it to change this once packages are in.